src/katello/repos.py

```diff
@@ -0,0 +1,114 @@
+import os
+import sys
+
+sys.path.append('/usr/lib/yum-plugins')
+sys.path.append('/usr/share/rhsm')
+
+try:
```

Comment:
I'd suggest making a `compat.py` that does these imports, and then `from compat import ConsumerIdentity` so you can use this in other files (like src/katello/tracer)